src/main/scala/io/sdkman/repos/VersionsRepo.scala

```diff
@@ -15,8 +15,7 @@ trait VersionsRepo {
 
   def findAllVersionsByCandidatePlatform(candidate: String, platform: String): Future[Seq[Version]] =
     versionsCollection
-      .find(and(
-        equal("candidate", candidate),
+      .find(and(equal("candidate", candidate),
```

Comment:
@marc0der Is this what you envisioned?

Reply:
Was thinking
```
find(
   and(…
   or(…
   or(…
```

Apologies for my OCD 😜

Reply:
I can totally relate to that 😄 !